src/components/ShoppingList/ShoppingList.js

```diff
@@ -62,6 +62,90 @@ function ShoppingList({ listId }) {
     setFilter(e.target.value);
   };
 
+  const getDaysToPurchase = (item) => {
+    let nextPurchaseDate;
+    if (item.lastPurchaseDate?.seconds) {
+      // if the item has been purchased before, next purchase date is [purchaseInterval] days from the lastPurchaseDate
```

Comment:
This is super minor, but for my understanding, why do you put purchaseInterval in brackets throughout your comments? Is it denoting something specific about purchaseInterval?

Reply:
@connietran-dev I think we were just trying to be clear that we were using it to indicate that it had a kind of dynamic value in the sentence, but I can see how it could actually be confusing mixed with code where [ ]'s mean something specific. Do you have a suggestion for what would read clearer to you? Because I'd be happy to change it to improve comment readability.

Reply:
I think when I first saw purchaseInterval with brackets, I thought it was an array or maybe an object's property name. I would simply remove it or put it around backtickets to indicate it's code / a variable name. Thanks for being open to suggestion!